### DISCUSSION TITLE
blog/realtime-chat-application-with-blazor/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#32] [General]
# blog/realtime-chat-application-with-blazor/

In this Guide, we will be building a full-fledged Chat Application With Blazor WebAssembly using Identity and SignalR from scratch.

https://codewithmukesh.com/blog/realtime-chat-application-with-blazor/

**Comment by rogercomp** [comment]
When I go to publish the client part I am getting an error with the following message - "Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Could not load type of field 'Web.Layout.MainLayout:<hubConnection>k__BackingField' (4) due to: Could not load file or assembly 'netstandard, Version=2.1.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies.
System.TypeLoadException: Could not load type of field 'Web.Layout.MainLayout:<hubConnection>k__BackingField' (4) due to: Could not load file or assembly 'netstandard, Version=2.1.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies.
   at Microsoft.AspNetCore.Components.Rendering.ComponentState.RenderIntoBatch(RenderBatchBuilder , RenderFragment , Exception& )" locally it works normally

**Reply by rogercomp** [reply]
any ideia?

**Reply by rogercomp** [reply]
publishing to nginx

**Reply by rogercomp** [reply]
Blazor Wasm NET 8
